Ginger/GingerCoreCommon/VariablesLib/VariableDateTime.cs

```diff
@@ -82,7 +82,7 @@ public string InitialDateTime
             }
             set
             {
-                if (!CheckDateTimeWithInRange(value))
+                if (mMinDateTime != null && mMaxDateTime == null && !CheckDateTimeWithInRange(value))
```

Comment:
Should we also add null check for mMinDateTime as you added for mMaxDateTime?

Reply:
done